projects/admin-core/src/lib/application/application-layer-settings/application-layer-settings.component.html

```diff
@@ -28,4 +28,8 @@ <h2>Settings for {{ node.label }}</h2>
     <mat-label>Attribution</mat-label>
     <input matInput formControlName="attribution" />
   </mat-form-field>
+  <mat-form-field>
+    <mat-label>Description</mat-label>
+    <input matInput formControlName="description" />
```

Comment:
Maybe make this a multi-line textarea.

Ideally we would also have a preview of the `htmlify` pipe or some documentation.